### ISSUE TITLE
Missing characters in TXT value in output

### ISSUE THREAD

**Issue by Prashant-fc** [blechschmidt/massdns#100]
Value of TXT record has some of the starting characters trimmed.

**Comment by srknzl** [comment]
* I confirm this issue exists
* first characters of dns values are replaced with \"

command run: `massdns -r resolvers.txt -t TXT -w j.txt a.txt`

contents of a.txt 
```
misli.com
```
output:
```
;; Server: 77.88.8.8:53
;; Size: 279
;; Unix time: 1611578286
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 43766
;; flags: qr rd ra ; QUERY: 1, ANSWER: 4, AUTHORITY: 0, ADDITIONAL: 0

;; QUESTION SECTION:
misli.com. IN TXT

;; ANSWER SECTION:
misli.com. 3600 IN TXT "=spf1 ip4:91\x2e93\x2e121\x2e190 ip4:91\x2e93\x2e121\x2e166 ip4:91\x2e93\x2e121\x2e86 include:spf\x2eprotection\x2eoutlook\x2ecom -all" 
misli.com. 3600 IN TXT "S=ms67801968" 
misli.com. 3600 IN TXT "AMOOD6PU/1LRmzUdApNvBHiCsjLNa6K1QYw6ydJDSo=" 
misli.com. 14400 IN TXT "XxUhFpg/R2R2RB32suaXm6P2KRdDGrTcPASijDPVE8=" 




```

**Comment by blechschmidt** [comment]
A quick fix is implemented by d77c68bf2ac37e520dbd08361d18218da2eb02b5, which fixes missing characters. Issue #103 will be fixed in the future.
